pkg/apis/garden/types.go

```diff
@@ -177,10 +177,6 @@ type OpenStackProfile struct {
 	// DNSServers is a list of IPs of DNS servers used while creating subnets.
 	// +optional
 	DNSServers []string
-	// DHCPDomain is the dhcp domain of the OpenStack system configured in nova.conf. Only meaningful for
-	// Kubernetes 1.10.1+. See https://github.com/kubernetes/kubernetes/pull/61890 for details.
-	// +optional
-	DHCPDomain *string
```

Comment:
Don't remove the field entirely without a proper deprecation period. Mark it rather as "deprecated" and don't use it anymore anywhere.